release/release_tests.yaml

```diff
@@ -1010,6 +1010,47 @@
 
   alert: default
 
+#######################
+# Lightning tests
+#######################
+
+- name: lightning_trainer_test_latest
```

Comment:
Do we need the `latest` tag at the end? What is `latest`: Ray, PTL?

What about this naming pattern:

```
lightning_gpu_3x16+1  <-- 3 x (16 CPU + 1 GPU)
lightning_gpu_tune_3x16+1  
```

This way, you can filter out tests based on prefix/suffix:

```
name:lightning_gpu_tune.*
name:lightning_gpu.*
```

Also, can add `cpu` only option in the future.

Reply:
And add a comment about the naming in the yaml